### PR TITLE
fix: Skipping wizard steps

### LINKED ISSUE #2998: Wizard steps can be skipped
### Package

filament/forms

### Package Version

v2.13.14

### Laravel Version

v9.18

### Livewire Version

v2.10.6

### PHP Version

PHP 8.16

### Bug description

Clicking rapidly on the Next button in a Wizard allows for steps skip validation. This probably isn't an issue if explicitly validating the entire form on the submit action but could have some unforeseen consequences for various use cases. Haven't looked into implications inside the Admin Panel just yet.

https://user-images.githubusercontent.com/3736774/176814543-a7be59e6-6a67-4b04-9967-5225a0b9ff49.mov

### Steps to reproduce

```php
protected function getFormSchema(): array
{
    return [
        Wizard::make([
            Wizard\Step::make('Step 1')
                ->schema([
                    TextInput::make('input-1')
                        ->required(),
                ]),
            Wizard\Step::make('Step 2')
                ->schema([
                    TextInput::make('input-2')
                        ->required(),
                ]),
            Wizard\Step::make('Step 3')
                ->schema([
                    TextInput::make('input-3')
                        ->required(),
                ]),
        ]),
    ];
}
```

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by zepfietje**

### CODE REVIEW

**Comment by danharrin** [comment]
Let's switch all occurrences of `wire:loading.attr.delay="disabled"`

**Comment by danharrin** [comment]
Yeah, its not a visual change like `wire:loading.class` so we dont need the delay

**Comment by zepfietje** [comment]
@danharrin, done!

File: packages/forms/resources/views/components/wizard.blade.php [hunk 1]
```diff
@@ -209,7 +209,6 @@ class="w-5 h-5"
                 x-show="! isLastStep()"
                 x-cloak
                 x-on:click="$wire.dispatchFormEvent('wizard::nextStep', '{{ $getStatePath() }}', getStepIndex(step))"
-                wire:loading.attr.delay="disabled"
```
Comment (danharrin) [inline packages/forms/resources/views/components/wizard.blade.php:212]: This line has been entirely removed?

Comment (zepfietje) [reply packages/forms/resources/views/components/wizard.blade.php:212]: Yeah, `<x-filament-support::button>` defines that attribute already. It was overridden previously to remove the `.delay` modifier. Now it's the same across those two components, so it no longer needs to be set on the button in the wizard.

**Pull request merged**